botfront/imports/ui/components/nlu/models/NLUTrainButton.jsx

```diff
@@ -22,12 +22,12 @@ class NLUTrainButton extends React.Component {
     }
```

Comment:
Shouldn't that component be renamed and put out of the nlu hierarchy? Coordinate with @MatthieuJnon

Reply:
Will coordinate with @MatthieuJnon

Reply:
I agree it's used everywhere now, @Palashx you can move that file to a more general folder

Reply:
cool will do it and make it a generic train button which can be later on used at any place